README.md

```diff
@@ -56,4 +56,17 @@ Then visit http://localhost:8100
 
 > If you want to re-run the `--build` command, you may get an error about not having access to delete a `cache/` folder. Use `sudo` to delete this manually before retrying.
 
+### Static Docker build
 
+Additionally, the `Dockerfile-standalone` Dockefile allows to produce a lightweight static build which also does not need local files to run. This can be useful in certain scenarios, for example for sharing the build output easily.
```

Comment:
> This can be useful in certain scenarios, for example for sharing the build output easily.

I'm not sure I understand the use case.

Reply:
As the image does not depend on the local folder, you can `docker push` the image to serve it elsewhere if, for example, I wanted to share the site with someone I'm working with.

Probably not something you would do every day, but if you want to, this will help.